assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -63,25 +63,41 @@ function useCanvasKeys(ref) {
     }
   );
 
-  // Return focus back to the canvas when another section loses the focus.
+  /**
```

Comment:
This shouldn't be a jsdoc, since it has no typing/declaration info. Is it some tool that replaced `//` with jsdoc? Or just personal preference?

Reply:
Oh just personal preference. Happy to revert back to `//` knowing we're only using block comments as jsdocs

Reply:
You can still use block comments. It just can't have those two asterisks. E.g. it's `/*`, and not `/**`.